ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -184,3 +184,19 @@ spec:
 #                user:
 #                  name: root
 #     ---- sources needed for updating CRI-O workload-partitioning ends here ----
+
+    # ---- Sources needed to configure IPsec encryption for external traffic ----
+    # 1. Set the `$mode` in the `enable-ipsec.yaml`
```

Comment:
if we want to be strict, can hard code this to `External`
and then add `Full` option once we discover there an actual telco use case for it